tests/unit/schema/customScalars.test.ts

```diff
@@ -0,0 +1,52 @@
+import {NonEmptyString} from '../../../src/schema/customScalars';
+describe('customScalars Unit Test', () => {
+    describe('NonEmptyString.serialize', () => {
+      test('Should return error for non-string value', async () => {
+        try {
+          var result = NonEmptyString.serialize(23);
+        } catch (error: any) {
+          expect(error.name).toStrictEqual("WRONG_VALUE_TYPE");
+        }
+      });
+
+      test('Should return error for empty string value', async () => {
+        try {
+          var result = NonEmptyString.serialize('');
```

Comment:
I can see by the title that the test is expecting an error, but if this line doesn't throw anything it will pass the test anyway

Reply:
Yeah, you are right I skipped that case where if someone stop throwing the error from subject method then it will pass anyway. I should have used following instead of current expect statementexpect(NonEmptyString.serialize('')).toThrowError({message: `Value can't be empty`, name: 'VALUE_NOT_EMPTY'});